go.mod

```diff
@@ -54,36 +49,30 @@ require (
 	github.com/googleapis/gax-go/v2 v2.0.5 // indirect
 	github.com/googleapis/gnostic v0.3.0 // indirect
 	github.com/gophercloud/gophercloud v0.2.0 // indirect
-	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/grpc-ecosystem/grpc-gateway v1.9.2 // indirect
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hpcloud/tail v1.0.0
 	github.com/imdario/mergo v0.3.7 // indirect
-	github.com/ishidawataru/sctp v0.0.0-20180213033435-07191f837fed // indirect
 	github.com/json-iterator/go v1.1.6
 	github.com/kisielk/errcheck v1.2.0 // indirect
 	github.com/klauspost/compress v1.7.1 // indirect
 	github.com/klauspost/cpuid v1.2.1 // indirect
-	github.com/klauspost/pgzip v1.2.1 // indirect
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/mailru/easyjson v0.0.0-20190620125010-da37f6c1e481 // indirect
 	github.com/mattn/go-isatty v0.0.8 // indirect
 	github.com/mattn/go-shellwords v1.0.5 // indirect
-	github.com/mistifyio/go-zfs v2.1.1+incompatible // indirect
 	github.com/mrunalp/fileutils v0.0.0-20171103030105-7d4729fb3618
-	github.com/mtrmac/gpgme v0.0.0-20170102180018-b2432428689c // indirect
 	github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30 // indirect
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opencontainers/runc v1.0.0-rc6
-	github.com/opencontainers/runtime-spec v0.0.0-20181111125026-1722abf79c2f
-	github.com/opencontainers/runtime-tools v0.9.0
+	github.com/opencontainers/runtime-spec v1.0.1
+	github.com/opencontainers/runtime-tools v0.8.0
```

Comment:
This used to be 0.9.0; why downgrade? Also, further down the hash for version 0.3.1 is added.

Reply:
Good catch. This comes from buildah's go module which is using `v0.8.0`.

@rhatdan, can you revert to using `v0.9.0` again?

Reply:
Nalin just opened a PR for updating Buildah, which hopefully will fix this.  Once we have it vendored, we can cut buildah 1.9.2 and update this PR.